core/arch/arm/kernel/entry_a64.S

```diff
@@ -56,11 +56,20 @@
 	.endm
 
 FUNC _start , :
+#if defined(CFG_CORE_SEL1_SPMC)
+	/*
+	 * With OP-TEE as SPMC at S-EL1 the SPMD (SPD_spmd) in TF-A passes
+	 * the DTB in x0 and the rest of the registers are unused
```

Comment:
`x1` is pageable part

What about the a32 implementation? Shouldn't `r1` play that role too?

Reply:
You're right. But AArch32 hasn't been tested with FF-A yet, I don't have a working prototype for that. I've just made the corresponding changes.

Reply:
Sounds reasonable.
Back to the a64 code, this comment still needs to be updated "DTB in x0 and the rest of the registers are unused", wrong, x1 is used.

Reply:
Sorry, I'll fix.